core/src/main/resources/filodb-defaults.conf

```diff
@@ -613,7 +626,7 @@ gateway {
 
   # Maximum size of queue for each shard's records.  Must be a power of 2.  When the max queue size is reached
   # then the connection will stop accepting incoming records.
-  max-queue-size = 16384
+  max-queue-size = 262144
```

Comment:
quick question for my understanding: why are we increasing the max-queue-size 16 times at the gateway ?

Reply:
My M1-mac was filling the queues too quickly before the Kafka-writer could write them to the disk. Hence needed a larger queue.